### PR TITLE
[INLONG-2862][Manager] Support to start Sort tasks through Flink

### LINKED ISSUE #2862: [Feature][Manager] Startup sort task through the ordinary flink cluster
### Description

update the sort task Through ordinary flink cluster

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Title Name: [INLONG-MANAGER][manager] manager startup sort task by flink

where *XYZ* should be replaced by the actual issue number.

Fixes #2862

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-plugins/pom.xml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    <artifactId>manager-plugins</artifactId>
+    <name>Apache InLong - Manager Plugins</name>
+
+    <properties>
+        <project.build.encoding>UTF-8</project.build.encoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <lombok.version>1.18.20</lombok.version>
+        <plugin.compile.version>3.8.1</plugin.compile.version>
+        <in.version>1.1.0-incubating-SNAPSHOT</in.version>
```
Comment (dockerzhang) [inline inlong-manager/manager-plugins/pom.xml:39]: useless, remove this version.

File: inlong-manager/manager-plugins/pom.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>manager-client</artifactId>
+            <version>1.1.0-incubating-SNAPSHOT</version>
```
Comment (dockerzhang) [inline inlong-manager/manager-plugins/pom.xml:88]: use <version>${project.version}</version>

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    public static final String SINK_INFO = "sink_info";
+
+    public static final String TYPE = "type";
+
+    public static final String ENTRYPOINT_CLASS = "org.apache.inlong.sort.singletenant.flink.Entrance";
+
+    public static final String INLONG = "INLONG_";
+
+    public static final String SORT_JAR = "sort-single-tenant-1.1.0-incubating-SNAPSHOT.jar";
```
Comment (dockerzhang) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java:42]: here should not use the fixed version.

File: inlong-manager/manager-plugins/src/main/assembly/package.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dockerzhang) [inline inlong-manager/manager-plugins/src/main/assembly/package.xml:1]: can we reuse the assembly.xml in manager-web？

File: inlong-manager/manager-plugins/pom.xml (hunk 1 above)
Comment (dockerzhang) [inline inlong-manager/manager-plugins/pom.xml:39]: this line is useless.

File: inlong-manager/manager-plugins/pom.xml (hunk 2 above)
Comment (dockerzhang) [inline inlong-manager/manager-plugins/pom.xml:88]: <version>${project.version}</version>

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java (hunk 3 above)
Comment (dockerzhang) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java:42]: fixed version is not good.

File: inlong-manager/manager-plugins/pom.xml (hunk 2 above)
Comment (LvJiancheng) [reply inlong-manager/manager-plugins/pom.xml:88]: done

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java (hunk 3 above)
Comment (LvJiancheng) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java:42]: done

File: inlong-manager/manager-plugins/pom.xml (hunk 2 above)
Comment (LvJiancheng) [reply inlong-manager/manager-plugins/pom.xml:88]: done

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java (hunk 3 above)
Comment (LvJiancheng) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java:42]: yeah ,through Regular matching

File: inlong-manager/manager-plugins/pom.xml (hunk 1 above)
Comment (LvJiancheng) [reply inlong-manager/manager-plugins/pom.xml:39]: ok

File: inlong-manager/manager-plugins/src/main/assembly/package.xml (hunk 4 above)
Comment (LvJiancheng) [reply inlong-manager/manager-plugins/src/main/assembly/package.xml:1]: reuse same assembly.xml ,manager-plugins can not package independently

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java (hunk 3 above)
Comment (dockerzhang) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/Constants.java:42]:
```suggestion
```

**Review by EMsnap — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**